### ISSUE TITLE
Fail if Meteor can't start in the start.js script

### ISSUE THREAD

**Issue by gbhatnag** [xolvio/automated-testing-best-practices#29]
I'm new to automated testing with Chimp and am having issues seeing the actual tests that are part of this package.  Specifically, when I run `npm run karma` or `npm test`, I do not see any of the tests listed and their results.  I'm expecting to see a list like I do with Mocha (that's what I'm most used to).  I do not see the list in the Terminal or in browser at http://localhost:9876.  There is an error, `Some of your tests did a full page reload!`, that seems to stop the tests from running...

Here's a snippet of what I'm running and seeing in the terminal:

```
$ npm test

> @ test /Users/gauravb/code/automated-testing-best-practices
> .scripts/test.js

22 03 2016 17:01:37.246:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
22 03 2016 17:01:37.253:INFO [launcher]: Starting browser Chrome
22 03 2016 17:01:37.915:INFO [Chrome 49.0.2623 (Mac OS X 10.11.0)]: Connected on socket /#UM5DYBkrknB6sAymAAAA with id 99027998
22 03 2016 17:01:38.399:WARN [web-server]: 404: /merged-stylesheets.css?hash=10624882b9f7b639d9b1eef9287a239f4e4f800c
Chrome 49.0.2623 (Mac OS X 10.11.0) ERROR
  Some of your tests did a full page reload!


Karma exited with code 1
npm ERR! Test failed.  See above for more details.
```

I'm very excited to get automated testing included in my Meteor app and Chimp seems like a great platform.  Any help in getting things up and running as soon as possible would be greatly appreciated!

Here's some environment info that may be of use:
`node v0.12.7 (npm v2.11.3)`
`Meteor 1.2.1`
`Mac OS X 10.11`

**Comment by lgandecki** [comment]
I've had a similar problem when running a meteor server in the background. Could you double check that you don't have any meteor apps running?

**Comment by gbhatnag** [comment]
Good call -- no Meteor apps running in the background seems to do the trick.

**Comment by SamHatoum** [comment]
We should probably do a check if Meteor complains that another app is already running in the startup script

**Comment by gbhatnag** [comment]
Yep -- a more descriptive/accurate error message would surely alleviate this for those who encounter this in the future.
